pallets/identity/src/claims.rs

```diff
@@ -536,4 +547,24 @@ impl<T: Config> Module<T> {
             );
         });
     }
+
+    pub fn base_register_custom_claim_type(origin: T::Origin, ty: Vec<u8>) -> DispatchResult {
+        let did = Self::ensure_perms(origin)?;
+        let id = Self::unsafe_register_custom_claim_type(ty.clone())?;
+        Self::deposit_event(Event::<T>::CustomClaimTypeAdded(did, id, ty));
+        Ok(())
+    }
+
+    fn unsafe_register_custom_claim_type(ty: Vec<u8>) -> Result<CustomClaimTypeId, DispatchError> {
```

Comment:
It looks like nothing is calling the `unsafe_*` function, so these two can be merged.